src/app/components/Age/__tests__/index.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { screen } from '@testing-library/react'
+import { renderWithProviders } from '../../../utils/renderWithProviders'
+import { Age } from '../'
+
+describe('Age', () => {
+  beforeEach(() => {
+    jest.useFakeTimers()
+    jest.setSystemTime(new Date('2024-02-05T10:14:35.000Z'))
+  })
+
+  afterEach(() => {
+    jest.useRealTimers()
```

Comment:
I think `afterEach` is missing `runOnlyPendingTimers` call. According to docs here: https://testing-library.com/docs/using-fake-timers/

Reply:
not sure we need it here
https://jestjs.io/docs/timer-mocks#run-pending-timers